curbs/examples.md

```diff
@@ -158,33 +163,38 @@ A [Query Curb Policies](/curbs#query-curb-policies) example of `/curbs/policies`
         {
```

Comment:
In this third example, the `"data": [` array needs to be removed too.

Reply:
fixed in [942d9b0](https://github.com/openmobilityfoundation/curb-data-specification/pull/132/commits/942d9b05456e09bac16e00f20014e9b3bf4d9158)